client/jetpack-connect/site-url-input.jsx

```diff
@@ -87,7 +87,7 @@ class JetpackConnectSiteUrlInput extends PureComponent {
 		return (
 			<p className="jetpack-connect__tos-link">
 				{ this.props.translate(
-					'By setting up Jetpack you agree to our fascinating {{tosLinkText}}Terms of Service{{/tosLinkText}} ' +
+					'By setting up Jetpack you agree to our {{tosLinkText}}Terms of Service{{/tosLinkText}} ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'By setting up Jetpack you agree to our fascinating {{tosLinkText}}Terms of Service{{/tosLinkText}} and to sync {{syncLinkText}}certain data and settings{{/syncLinkText}} to WordPress.com' )` *ES Score: 26*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).